prow/cmd/crier/README.md

```diff
@@ -48,8 +48,17 @@ The actual report logic is in the [github report library](/prow/github/report) f
 
 You can enable the Slack reporter in crier by specifying the `--slack-workers=n` and `--slack-token-file=path-to-tokenfile` flags.
 
-The `--slack-token-file` flag takes a path to a file containing a Slack [*OAuth Access Token*](https://api.slack.com/docs/oauth). The access token can be obtained after installing 
-a [Slack app](https://api.slack.com/apps) for the *workspace*. The app should have the [`chat:write:bot`](https://api.slack.com/scopes/chat:write:bot) scope granted in order to [post messages](https://api.slack.com/methods/chat.postMessage) to the configured Slack *channel*.
+The `--slack-token-file` flag takes a path to a file containing a Slack [**OAuth Access Token**](https://api.slack.com/docs/oauth).
+
+The **OAuth Access Token** can be obtained as follows:
+1. Navigate to: https://api.slack.com/apps.
+2. Click **Create New App**; 
+3. Provide an **App Name** (e.g. Prow Slack Reporter) and **Development Slack Workspace** (e.g. Kubernetes).
+4. Click **Permissions**.
+5. Add the `chat:write:bot` scope using the **Scopes** dropdown and **Save Changes**.
+6. Click **Install App to Workspace**
+7. Click **Allow** to authorize the Oauth scopes.
+7. Copy the **OAuth Access Token** and save the token to a file.
```

Comment:
Duplicate step number. You can use `1. ` in front of all items and the correct number will be shown when redered.

Also maybe add a sample command for creating the secret, something like `kubectl create secret generic slack-token --from-literal=token=$TOKEN` ?